src/mod/internal/login_mod.cpp

```diff
@@ -70,7 +98,15 @@ LoginMod::LoginMod(
     }
     this->screen.add_widget(&this->login);
 
-    this->login.login_edit.set_text(username);
+
+    if (vars.get<cfg::internal_mod::enable_target_field>()) {
+        auto [target, login] = rpartition(std::string(username), ":+");
+        this->login.login_edit.set_text(login.c_str());
```

Comment:
`.c_str()` is useless ; `this->login.login_edit.set_text(std::move(login));` is best.

Reply:
edit widget `set_text` only accept null terminated const char*